src/Providers/Trustpilot/README.md

```diff
@@ -0,0 +1,60 @@
+# Trustpilot
+[API Specification](https://documentation-apidocumentation.trustpilot.com/)
+
+## Usage
+
+#### Create a trustpilot client
+Any PHP Application
+```
+$reviews = new \PodPoint\Reviews\Reviews();
+$trustpilot = $reviews->withProvider('trustpilot')
+```
+Laravel:
+```php
+$trustpilot = Reviews::trustpilot();
+```
+
+#### Send a merchant invite
+Required Fields:
+* consumerEmail
+* consumerName
+* referenceNumber
+
+```php
+$trustpilot->merchant()->invite([
+    'consumerEmail' => 'john.doe@trustpilot.com', 
+    'consumerName'=> 'John Doe',
+    'replyTo' => 'john.doe@trustpilot.com',
+    'referenceNumber' => 'inv00001',
+    'locale' => 'en-US',
+    'senderEmail' => 'john.doe@trustpilot.com',
+    'serviceReviewInvitation' =>  [
+        'preferredSendTime' => '2013-09-07T13:37:00',
```

Comment:
I think it's better that we enforce using our package this way because it matches the structure that we actually send out

this means that we need to adjust these values: 
![image](https://user-images.githubusercontent.com/58700660/95584548-d3ecfb80-0a35-11eb-9917-73043c6e18f0.png)